src/libs/intents/localMethods.spec.js

```diff
@@ -7,28 +7,35 @@ import { localMethods, asyncLogout } from './localMethods'
 
 jest.mock('react-native-keychain')
 jest.mock('../RootNavigation.js')
+jest.mock('@react-native-cookies/cookies', () => ({
+  clearAll: jest.fn()
+}))
 
 describe('asyncLogout', () => {
+  const client = {
```

Comment:
I think this whole file was a bad idea from the start. We should have one file per tested method, in .ts.
Maybe for this PR it is possible to have a `asyncLogout.spec.ts` and ignore the rest of the file?

Reply:
I would prefer to do this in another commit